build.rs

```diff
@@ -35,19 +35,29 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     fs_extra::copy_items(&source_paths, &out_dir, &copy_options)?;
 
     // Compile the C library.
-    let mut make = Command::new("make");
-
-    make.env_remove("PROFILE").arg("-C").arg(&out_dir).arg("build");
-
-    if env::var("PROFILE").unwrap() == "debug" {
-        make.arg("DEBUG=1");
+    let mut build = cc::Build::new();
+    build
+        .files(glob(out_dir.join("src/*.c").to_str().unwrap()).unwrap().map(|p| p.unwrap()))
+        .files(glob(out_dir.join("src/postgres/*.c").to_str().unwrap()).unwrap().map(|p| p.unwrap()))
+        .file(out_dir.join("vendor/protobuf-c/protobuf-c.c"))
+        .file(out_dir.join("vendor/xxhash/xxhash.c"))
+        .file(out_dir.join("protobuf/pg_query.pb-c.c"))
+        .include(out_dir.join("."))
+        .include(out_dir.join("./vendor"))
+        .include(out_dir.join("./src/postgres/include"))
+        .include(out_dir.join("./src/include"));
```

Comment:
Do we want to suppress the C warnings here, using [`.warnings(false)`](https://docs.rs/cc/latest/cc/struct.Build.html#method.warnings)? These might not be as relevant to the `pg_query.rs` project, compared to working directly in the `libpg_query` repo.